test/features/dataset.all.feature.orig

```diff
@@ -0,0 +1,258 @@
+@api
```

Comment:
@topicus looks like you need to configure your mergetool to stop backing up: `git config --global mergetool.keepBackup false`